Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,8 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics)
```

Comment:
Hi Giovanni,

if I remember, this was just to follow a similar nomenclature as in other configuration files.